lib/util/git/index.ts

```diff
@@ -96,6 +94,14 @@ function checkForPlatformFailure(err: Error): void {
       'GitLab: Commit message does not follow the pattern',
       "Cannot push because commit message does not follow project's push rules",
     ],
+    [
+      ' is not a member of team',
+      '`Restrict commits to existing GitLab users` rule is blocking renovate push. Check renovate `gitAuthor` setting',
+    ],
+    [
+      'TF401027:', // You need the Git 'GenericContribute' permission to perform this action
+      'You need the Git `GenericContribute` permission to perform this action',
```

Comment:
```suggestion
      'You need the Git `GenericContribute` permission to perform this action, do .... to fix the problem',
```

Maybe we can help users fix this by pointing them to the docs they need to fix this?
It feels a little unhelpful the way it is now. 😄

Reply:
@JamieMagee  Do you now how to fix the permission issue in azure devops? As this is a server side permission issue.